package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperswitch",
-  "version": "0.11.1",
+  "version": "0.12.0",
```

Comment:
No need to go to `v0.12.0`, bumping the version to `v0.11.2` is enough.

Reply:
Done.